src/api.ts

```diff
@@ -233,6 +233,7 @@ router.post(
         data: {
           ...newBid,
           auctionId: newBid.bidNonce, // compatibility with DSS
```

Comment:
Hmm. There's no handling of bids yet in the DSS. And this was renamed in the contracts because its more accurate.

I think we just bite the bullet here and update zns-message-schema to use `bidNonce` over auctionId, then update this api.ts here to also use bidNonce.

Reply:
Added!